lib/WebAudioRecorderMp3.js

```diff
@@ -39,7 +39,7 @@ function record(buffer) {
   if (bufferCount++ < maxBuffers)
     if (encoder)
       encoder.encode(buffer);
-    else
+    else if (recBuffers)
```

Comment:
This fix is wrong. You need to wrap the top level condition with brackets otherwise you will trigger a timeout with the following `else`.

```
    if (bufferCount++ < maxBuffers) {
        if (encoder)
            encoder.encode(buffer);
        else if (recBuffers)
            recBuffers.push(buffer);
    } else
        self.postMessage({ command: "timeout" });
```